.github/workflows/static-code-analysis.yml

```diff
@@ -0,0 +1,16 @@
+name: Static code analysis
+
+on: [pull_request]
+
+jobs:
+  static-code-analysis:
+    runs-on: ubuntu-latest
+    services:
+        volumes:
+            - .:/project
```

Comment:
I just did a quick test on one of my projects. You don't need to mount any directories, github will do it for you. 

This is all you need:

```
name: Static code analysis

on: [pull_request]

jobs:
  phpstan:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@master
      - name: Run PHPStan
        uses: docker://jakzal/phpqa:php7.3-alpine
        with:
          args: phpstan analyze src/

```

Reply:
Actually, I think to make it work jakzal/phpqa needs to be updated to run composer etc.